template-deliverables.md

```diff
@@ -0,0 +1,145 @@
+# Template deliverables
+
+This document explains the different documentation deliverables that template authors are required to write as part of each template set. First, read our Template Contributing Guide to understand the larger context for these deliverables.
+
+## Table of contents
+
+- [Overview](#overview)
+- [Raw template](#raw-template)
+- [Template guide](#template-guide)
+- [Before you start guide (optional)](#before-you-start-guide-optional)
+- [Template example (optional)](#template-example-optional)
+
+
+## Overview
+
+Each template set should have the following components:
+
+<table>
+  <tr>
+    <th>Deliverable</th>
+    <th>Description and purpose</th>
+    <th>Required?</th>
+  </tr>
+  <tr>
+    <td>Raw template</td>
+    <td>The raw template is the basic template for the type of document you are creating. It provides a rough outline of the suggested content and a few embedded writing tips for how to fill in the different sections of the template.</td>
+    <td>Yes, required</td>
+  </tr>
+  <tr>
+    <td>Template guide</td>
+    <td>This guide provides a deeper explanation of how to fill in the raw template. It provides a lightweight introduction to the purpose of this documentation and explains how to fill in each section of the document. Additional content may be included as needed.</td>
+    <td>Yes, required</td>
+  </tr>
+  <tr>
+    <td>Before you start guide</td>

```

Comment:
there may be some conceptual overlap here between Before you start and Guide.  If something needs to be understood before using the template, I'd think that should go in Guide. Stuff like references to support the decisions made in creating the template do not necessarily need to be understood "before you start". I had thought of this file more like a "Reference" or "About this template" file.

Reply:
See my previous comment.

Reply:
Resolved in our US general weekly meeting: We discussed this at length at last week's general meeting and the group consensus there was to call this the "deep dive" document. It basically contains miscellaneous information or guidance that is too comprehensive for the "guide" document. Based on whatever the each unique template needs, this document could include research, deep theories, pre-writing information, and/or any other material that users need to know when implementing this topic. Template writers are encouraged to point/link readers to that document in their guide to encourage users to read it in more depth.